contracts/lbp/LBPWrapper.sol

```diff
@@ -167,6 +167,69 @@ contract LBPWrapper {
         vault.joinPool(lbp.getPoolId(), address(this), address(this), request);
     }
 
+    /**
+     * @dev exit pool or remove liquidity from pool
+     * @param _tokens                   array of tokens sorted for the LBP
+     * @param _receiver                 address who will be credited with the tokens after removing liquidity
+     * @param _minAmountsOut            array of minimum amounts out
+     * @param _toInternalBalance        fund tokens to the internal user balance
+     * @param _userData                 userData specifies the type of exit
+     */
+    function removeLiquidity(
+        IERC20[] memory _tokens,
+        address payable _receiver,
+        uint256[] memory _minAmountsOut,
+        bool _toInternalBalance,
```

Comment:
I wasn't at the conversation but do we want to give the option to exit to `internalBalance`?

Reply:
not now.

Reply:
So are we not then going to remove it from the arguments? Like we did with the `fundingPool()`

Reply:
yes